test/jdk/sun/tools/jstatd/JstatdTest.java

```diff
@@ -322,8 +322,9 @@ private void runTest(boolean useShortSyntax) throws Throwable {
         }
 
         ProcessThread jstatdThread = null;
+        int tries = 0;
         try {
-            while (jstatdThread == null) {
+            while (jstatdThread == null && ++tries <= 10) {
```

Comment:
maybe use a ubiquitous constant MAX_RETRY (private static final in MAX_RETRY = 10)

Reply:
Yes, agreed.